test/unit-test/cmock/project.yml

```diff
@@ -22,10 +22,13 @@
     - <stdbool.h>
     - <stdint.h>
     - <fcntl.h>
-    - "FreeRTOS.h"
-    - "FreeRTOS_IP.h"
-    - "FreeRTOS_IP_Private.h"
-    - "FreeRTOS_UDP_IP.h"
+    - <unity.h>
```

Comment:
Is unity still needed? I saw earlier that it was removed from a file.

Reply:
Yes, here it is needed... :)